crates/rune/src/any.rs

```diff
@@ -72,7 +70,16 @@ pub use rune_macros::Any;
 ///     name: String,
 /// }
 /// ```
-pub trait Any: Named + any::Any {
+pub trait Any: Named + TypeHash + 'static {}
```

Comment:
This is leftover from when I was still getting my bearings on how all this worked. There's no need to drop `any::Any` from the trait, so this can be reverted.

Reply:
All right, please do once you have the time!